### PR TITLE
Grow 439

### PR DESCRIPTION
**Pull request by BoulderBrains**
Sponsors section of the IWD 2021 homepage

Content served from [contentful](https://app.contentful.com/spaces/j0p9a6ql0rn7/environments/development/entries/1mCe3WtDGvXsbMjIxftAsg?previousEntries=3RyBOMD5fjid9CtpAmYze3,7oYYUSWjeQdlnSUuuOHx70) 

**Desktop:**
Screen Shot 2021-02-19 at 4 25 05 PM

**Phone:**
Screen Shot 2021-02-19 at 4 24 46 PM

### CODE REVIEW

File: src/pages/Homepage/iwd/IWD2021Homepage.vue [hunk 1]
```diff
@@ -48,8 +51,12 @@ export default {
 			return this.content?.page?.contentGroups?.homepageHowItWorks ?? null;
 		},
 		midrollContentGroup() {
+			console.log('content groups', this.content?.page?.contentGroups);
```
Comment (mcstover) [inline src/pages/Homepage/iwd/IWD2021Homepage.vue:54]: please remove

**Review by mcstover — APPROVED** [review]

**Review by emuvente — COMMENTED** [review]
Visa is pretty strict about how their logo is displayed, so I'm going to be a little nit-picky:

- The IWD logo needs to have `padding: rem-calc(40);` and `max-width: rem-calc(320)` for the breakpoints smaller than `large`
- The logos need to be in a single column for the breakpoints smaller than `large`
- The margins between the logos need to be 64px apart for breakpoints smaller than `large` (`margin: rem-calc(32)`) and the logos need to be 128px apart for `large` and up (`margin: rem-calc(64)`)

**Comment by BoulderBrains** [comment]
Latest commit addresses adjustments mentioned previously. 

Desktop: 
Screen Shot 2021-02-19 at 4 25 05 PM

Mobile: 
Screen Shot 2021-02-22 at 2 49 46 PM

**Review by emuvente — APPROVED** [review]
Looks great! 👍

**Review by mcstover — APPROVED** [review]
Nice updates! Good call @emuvente

**Pull request merged**